docs/api-gateway/docs/06-01-api.md

```diff
@@ -35,12 +35,24 @@ This table lists all the possible parameters of a given resource together with t
 | Field   |      Mandatory      |  Description |
 |----------|:-------------:|------|
 | **metadata.name** |    **YES**   | Specifies the name of the exposed API |
-| **service.name**, **service.port** | **YES** | Specifies the name and the communication port of the exposed service. |
+| **spec.service.name**, **spec.service.port** | **YES** | Specifies the name and the communication port of the exposed service. |
 | **spec.hostname** | **YES** | Specifies the service's external inbound communication address. |
+| **spec.authenticationEnabled** | **NO** | Controls the authentication attributes of the Istio Authentication Policy that secures the service. See the usage-of-authenticationEnabled-parameter section. |
+| **spec.disableIstioAuthPolicyMTLS** | **NO** | Valid only if authentication is enabled. If `true`, disables the mTLS feature of the Istio Authentication Policy that secures the service. Defaults to `false`. |
 | **spec.authentication** | **NO** | Allows to specify an array of authentication policies that secure the service. |
 | **authentication.type** | **YES** | Specifies the desired authentication method that secures the exposed service. |
 | **authentication.jwt.issuer**, **authentication.jwt.jwksUri** | **YES** | Specifies the issuer of the tokens used to access the services, as well as the JWKS endpoint URI. |
 
+### Usage of authenticationEnabled parameter
+
+The `spec.authenticationEnabled` parameter controls if, and how, the API-Controller creates an Istio Authentication Policy that secures the exposed service.
+If authentication is enabled, `spec.authenticationEnabled` parameter sets conditions for the value of `origins` attribute of the corresponding Istio Authentication Policy.
```

Comment:
This sentence is redundant considering the bullets below and the information they give. Please remove it.